__tests__/lint/enforcements-all-rules-warn/.redocly.yaml

```diff
@@ -0,0 +1,34 @@
+apiDefinitions:
+  main: ./openapi.yaml
+
+lint:
+  rules:
+    enforcements:
+      - on:
+          - PathItem.Operation.summary
+        description: Operation summary example warn
+        severity: warn
+        pattern: '/test example/'
+        enum:
+          - summary test example
+        defined: false
+        nonEmpty: false
+        length: 16
+        minLength: 20
+        maxLength: 10
```

Comment:
These aren't in the PR description of supported use cases.

Reply:
Updated, thnx.